build.assets/Dockerfile

```diff
@@ -102,8 +102,8 @@ ENV GOPATH="/go" \
 RUN go install github.com/google/addlicense@v1.0.0
 
 # Install meta-linter.
-RUN (curl -L https://github.com/golangci/golangci-lint/releases/download/v1.38.0/golangci-lint-1.38.0-$(go env GOOS)-$(go env GOARCH).tar.gz | tar -xz && \
-     cp golangci-lint-1.38.0-$(go env GOOS)-$(go env GOARCH)/golangci-lint /bin/ && \
+RUN (curl -L https://github.com/golangci/golangci-lint/releases/download/v1.44.0/golangci-lint-1.44.0-$(go env GOOS)-$(go env GOARCH).tar.gz | tar -xz && \
```

Comment:
We will have to re-spin the buildbox, or update the linter script, in order for this update to make it onto CI.

Reply:
Yeah I figured there would be more involved than this. But I was constantly getting errors that aren't picked up by CI and believe it is due to the older version.

Reply:
I have PR #10179 that will remove some of the manual buildbox image management for GCB

Reply:
nit: I think the comment here is out of date. If I had to guess, at some point we switched from `go-metalinter` to `golangci-lint` and left the old install comment.